l10n_fr_siret_lookup/static/description/index.html

```diff
@@ -0,0 +1,477 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion

```
This line should be removed otherwise you get a warning error when installing the module : 
`WARNING odoo17 py.warnings: /home/user/dev/odoo17/ocb/odoo/addons/base/models/ir_module.py:178: DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, l10n_fr_siret_lookup can just have a UTF8 description with not need for a declaration.`

Note : this was fixed in maintainer-tools with https://github.com/OCA/maintainer-tools/pull/596 back in December 23 so it should not be present anymore when pre-commit reissues the index.html

Reply:
@David-Luis-Mora please can you make this correction so that the module in version 17.0 is available to the community?